zip-1014.rst

```diff
@@ -0,0 +1,439 @@
+* 25% for the Zcash Foundation, for general use (denoted **ZF-GU slice**);
+* 40% for additional "Major Grants" for large-scale long-term projects
+  (denoted **ZF-MG slice**).
+
+Details below. The fund flow will be implemented at the consensus-rule layer,
+by sending the corresponding ZEC to the designated address in each block. This
+Dev Fund will end at the second halving (unless extended/modified by a future
+ZIP).
+
+
+ECC slice (Electric Coin Company)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This slice of the Dev Fund will flow to ECC.
+
+ECC must undertake a firm obligation to use the Dev Fund only in support of the
+Zcash cryptocurrency and its community.
+
+In particular, ECC must commit to not distribute the Dev Fund proceeds to its
+partners ("shareholders"), other than:
+
+1. In fair-market-value compensation for specific new work (e.g., to employees
+   and contractors).
+2. For covering pass-through tax obligations to partners caused by ECC's receipt
+   of the Dev Fund.
+
+(ECC is encouraged to transition to a corporate structure that would avoid the
+latter taxes.)
+
+This obligation must be made irrevocable, e.g., within ECC's corporate
+governance structure (i.e., its Operating Agreement) or contractual obligations.
+
+
+ZF-GU slice (Zcash Foundation, for general use)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This slice of the Dev Fund will flow to ZF, to be used at its discretion for
+any purpose within its mandate to support Zcash and financial privacy,
+including: development, education, support community communication online
+and via events, gathering community sentiment, and external awarding grants
+for all of the above.
+
+
+ZF-MG slice (Zcash Foundation, for major grants)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This slice of the Dev Fund is intended to fund independent teams entering the
+Zcash ecosystem, to perform major ongoing development (or other work) for the
+public good of Zcash ecosystem, to the extent that such teams are available
+and effective.
+
+The funds will be received and administered by ZF. ZF will disburse them as
+"Major Grants", within the framework of ZF's grant program but subject to the
+following additional constraints:
+
+1. These funds may be only be used to issue Major Grants to external parties
+   that are independent of ZF. They may not be used by ZF for its internal
+   operations and direct expenses.
+
+2. Major Grants should support well-specified work proposed by the grantee,
+   at reasonable market-rate costs. They can be of any duration, or ongoing
+   without a duration limit, but have semiannual review points for
+   continuation of funding.
+
+3. Major Grants may be issued to ECC only if there are no other proposals
+   to perform the specified work with similar capabilities, effectiveness and
+   cost. The ECC will not be eligible for Major Grants if they have any funds
+   available in their slice's Volatility Reserve. (The intent is that
```

Comment:
Reiterating my comment (speaking for myself) at https://forum.zcashcommunity.com/t/community-sentiment-polling-results-nu4-and-draft-zip-1014/35560/224 :

The sentence "The ECC will not be eligible for Major Grants if they have any funds available in their slice's Volatility Reserve." is an addition [relative to ZIP 1012](https://github.com/zcash/zips/blob/master/zip-1012.rst#zf-mg-slice-zcash-foundation-for-major-grants).

This introduces a couple of problems:

* There is a perverse incentive for ECC to drain the Volatility Reserve in order to be able to apply for Major Grants (especially, but not only, if the amount in the Volatility Reserve is fairly small compared to the potential grant allocation).
* If the short-term coin price spikes so that a Volatility Reserve is created, then ECC is immediately excluded from applying for Major Grants — even if plans to make a grant application were in progress. In other words, ECC is effectively penalized for the coin price going up!

IMHO these are serious problems and the extra sentence should be removed.

Reply:
(This would be a natural place for a [multi-line suggestion](https://github.com/isaacs/github/issues/1452), but GitHub doesn't support those yet.)

Reply:
agree @daira @tromer  and removing it now, will update with a commit shortly